.github/workflows/ci.yml

```
@@ -56,7 +56,7 @@ jobs:
         if: contains(fromJson('["1", "1.9"]'), matrix.version) && matrix.os == 'ubuntu-latest'
       - uses: julia-actions/julia-processcoverage@v1
         if: contains(fromJson('["1", "1.9"]'), matrix.version) && matrix.os == 'ubuntu-latest'
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
```

Hi @CarloLucibello, codecov-action v4 mandates passing the codecov token in the CI. The coverage workflow will silently fail if the token is not present - https://github.com/FluxML/Flux.jl/actions/runs/7791697841/job/21248294001. See the instructions for generating and adding the token - https://docs.codecov.com/docs/adding-the-codecov-token.